src/documents/ModelRelay.md

```diff
@@ -4,33 +4,58 @@ layout: default
+The Infusion Model Relay system is a powerful scheme for supplying declarative configuration which specifies rules for keeping multiple pieces of model state around the component tree automatically up to date with each other's changes. 
+We sometimes use the term _**model skeleton**_ to refer to a connected set of models around the component tree which are related together by relay rules. This appeals to the idea that these models form an inner core which
+arrive at a set of mutually consistent model values, before they start to notify listeners out in the periphery of each component (generally, in its view layer). The system might make several passes over the model skeleton in
+order to satisfy all the relay rules, and then update all [`modelListeners`](ChangeApplierAPI.md#model-listener-declaration) as a single operation &#8212; presenting them with a consistent snapshot of the state of the entire
+skeleton all at once. This guarantees that no model listener ever sees a model value which is inconsistent with the relay rules which are meant to be keeping them in step, which greatly enhances the reliability and clarity of the
+application design.
+
+Every Infusion component descended from the grade `fluid.modelComponent` (a model-bearing component) supports a configuration area named `modelRelay` in which these rules can be defined.
 
 ### Two Styles for Configuring Model Relay ###
 
-There are two primary styles of configuration for setting up model relay - firstly, using the _implicit syntax_ which just consists of [IoC References](IoCReferences.md) from the model configuration for one model-bearing component to another - that is, in the component's configuration under the top-level model entry. Secondly, the _explicit syntax_ involves an entry in the component's top-level `modelRelay` entry expressing a more complex rule, most likely involving some form of [Model Transformation](to-do/ModelTransformation.md) to apply during the relay process. Both of these styles will set up a permanent and bidirectional relationship between the two models at the ends of the relay - the relationship will be established as the component(s) construct (during the _[initial transaction](#the-initial-transaction)_), and will persist until one of the components at the endpoints is destroyed.
+There are two primary styles of configuration for setting up model relay &#8212; firstly, using the _implicit syntax_ which just consists of [IoC References](IoCReferences.md) from the model configuration for one model-bearing component to another &#8212;
```

Comment:
These two styles might be better presented as a bulleted or numbered definition list.